builtin/providers/aws/resource_aws_autoscaling_group_test.go

```diff
@@ -262,7 +264,7 @@ resource "aws_launch_configuration" "foobar" {
 
 resource "aws_autoscaling_group" "bar" {
   availability_zones = ["us-west-2a"]
-  name = "foobar3-terraform-test"
+  name = "foobar3-terraform-test-%d"
```

Comment:
Ah, there is a test step above that tests this name directly. That is what @pearkes is talking about. Do the ASG acceptance tests pass?